src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/translator.kt

```diff
@@ -46,6 +45,9 @@ fun <P:Prisoner> translate(prisoner : P, ob: OffenderBooking): P {
 
   // get the most serious offence for this booking
   prisoner.mostSeriousOffence = ob.offenceHistory?.firstOrNull{ off -> off.mostSerious && off.bookingId == ob.bookingId }?.offenceDescription
+  prisoner.recall = ob.recall
+  prisoner.legalStatus = ob.legalStatus
+  prisoner.indeterminateSentence = ob.sentenceTerms?.any{ st -> true == st.lifeSentence && st.bookingId == ob.bookingId }
```

Comment:
this reads a bit odd with the double equals on `true`, had to double check it wasn't javascript!
Would have thought that
```kotlin
  prisoner.indeterminateSentence = ob.sentenceTerms?.any{ it.lifeSentence && it.bookingId == ob.bookingId }
```
would be better

Reply:
yes not sure why I did that!